py/sigver/test/step.py

```diff
@@ -0,0 +1,67 @@
+# SPDX-FileCopyrightText: 2023 Technology Innovation Institute (TII)
+# SPDX-License-Identifier: Apache-2.0
+""" Signature Verification Script """
+import argparse
+
+import logging
+
+import sys
+import os
+
+import json
+import requests
+
+LOG = logging.getLogger(os.path.abspath(__file__))
+
+
+# Default certificate to be used if no argument is given
+CERTIFICATE_NAME="INT-Ghaf-Devenv-Common"
+
+# Azure Function (verify-signature) URL
+URL = "https://ghaf-devenv-microsign-aleksandrtserepo-app.azurewebsites.net/api/verify-signature"
+#URL = "http://localhost:7071/api/VerifySignature"
+#URL = "https://ghaf-devenv-signverify.azurewebsites.net/api/verifysignature"
```

Comment:
Do these commented out lines serve any purpose? They're ok for me, if one expects to regularly enable the lines e.g. when debugging.